modin/pandas/general.py

```diff
@@ -454,6 +459,14 @@ def concat(
         ).index
     else:
         new_idx = None
+
+    if len(list_of_objs) == 0:
+        return DataFrame(
```

Comment:
`Codecov` shows that this part of the code is not tested. Is it true or not?

Reply:
It's true, because the test executes inner join, in which case the frames are note filtered. Should we add outer join either?

Reply:
Let's add and we can merge it.

Reply:
Done.